### ISSUE TITLE
YouTube Block: Video won't play via Apple Voiceover on iOS

### ISSUE THREAD

**Issue by kathrynwp** [Automattic/wp-calypso#59193]
### Quick summary

A WordPress.com user reports an accessibility bug with the YouTube block, whereby the video cannot be played with the usual manipulation.

### Steps to reproduce

1. Add a YouTube block to a page or post
2. User Apple's Voiceover on an iOS mobile device such as an iPhone or iPad, and try to play the video on the front end

### What you expected to happen

The video should play when the screen reader software is used to initiate a double tap effect

### What actually happened

According to the user, 

> When a mobile screen reader user on the IOS platform double taps play button on a YouTube embed it doesn’t work. A double tap and hold fixes the issue on IOS but this isn’t expected behaviour. To the user without knowledge of the workaround, the play button can’t be pressed

### Context

Customer report:

chat: 23295084-hc
follow-up ticket: 4612288-zen

### Browser, operating system and other notes

iOS with Apple Voiceover screenreader

### Severity

Some (< 50%)

### Available workarounds?

Yes, difficult to implement

### Workaround details

> A double tap and hold fixes the issue on IOS but this isn’t expected behaviour. To the user without knowledge of the workaround, the play button can’t be pressed

**Comment by kosiew** [comment]
Hi @kathrynwp ,

Thanks for reporting this and the detailed steps.
I managed to reproduce this on my iPhone.

Triaging for developers' further investigation.
